env/production/locals.tf

```diff
@@ -14,6 +14,7 @@ locals {
     "avian-flu"       = ["avian-flu"],
     "dengue"          = ["dengue"],
     "forecasts-ncov"  = ["forecasts-ncov"],
+    "hmpv"            = ["human-metapneumovirus"],
```

Comment:
FWIW, I did think we had an (informal) policy of trying to avoid creating more mismatches like this.

Reply:
https://github.com/nextstrain/nextstrain.org/issues/41

Reply:
Maybe the repo should be renamed?

Reply:
[Asking on Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1734045738381789?thread_ts=1734039097.017049&cid=C01LCTT7JNN)

Reply:
Repo has been renamed and updated repo name in https://github.com/nextstrain/infra/pull/34/commits/9cad7d4f1eee19da770b0b1ba19b0aafc8e354b3